servers/rendering/renderer_rd/effects/luminance.cpp

```diff
@@ -0,0 +1,255 @@
+/*************************************************************************/
```

Comment:
You'll have to update the copyright header now :(

Reply:
Done :)